tasks/less.js

```diff
@@ -125,15 +125,6 @@ module.exports = function(grunt) {
 
     var srcCode = grunt.file.read(srcFile);
 
-    // Equivalent to --modify-vars option.
-    // Properties under options.modifyVars are appended as less variables
-    // to override global variables.
-    var modifyVarsOutput = parseVariableOptions(options['modifyVars']);
```

Comment:
build is failing with jshint because you also need to remove parseVariableOptions